logrotate.go

```diff
@@ -284,10 +284,6 @@ func (l *Logger) openExistingOrNew(writeLen int) error {
 		return fmt.Errorf("error getting log file info: %s", err)
 	}
 
-	if info.Size()+int64(writeLen) >= l.max(int64(writeLen)) {
```

Comment:
lets say we set a size and the new check starts just at the border when the file is at max size but not yet at the tipping point, removing this check can potentially make the file not rotate?

Reply:
This function in called within the write function where there is another check to ensure that the logs don't overflow.
[if l.size+writeLen > l.max(writeLen)](https://github.com/fahedouch/go-logrotate/blob/6a8beddaea39b2b9c77109d7fa2fe92053c063e5/logrotate.go#L144)

Reply:
if `l.MaxBytes == -1` this cond should be always false => l.max(int64(writeLen)) should be equal to `MaxInt64`

```
		if l.MaxBytes == -1 {
			return MaxInt64
		}
```

removing it that means we lost rotating logic